students/geonwoo/users/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+# Create your models here.
+class Sign_up(models.Model):
```

Comment:
```suggestion
class User(models.Model):
```
class 이름은 어떤 데이터의 대한 정보인가를 나타내주어야 합니다.